repository.eselect.in

```diff
@@ -390,12 +390,24 @@ describe_list() {
 	echo "List all repositories"
 }
 
+describe_list_parameters() {
+	echo "[-I]"
+}
+
+describe_list_options() {
+	echo "-I         : Only list installed"
+}
+
 do_list() {
 	get_config
 	update_cache
 
 	write_list_start "Available repositories:"
 	local name state i=1
+	local only_installed
```

Comment:
Please move this to the beginning of the function, `shift` after taking it and use lowercase `-i`.

Reply:
Done